app/models/ticket_stakeholder.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class TicketStakeholder < ApplicationRecord
+  enum :stakeholder_type, {
+    user: "user",
+    user_group: "user_group",
+  }
```

Comment:
The only small downside to my polymorphic suggestion below: Rails does not support enums on polymorphic `*_type` columns. That's not your fault, it's a definition by Rails that somebody just said "This isn't supported". It's a shame, I would love to have it, but in this case I think the benefits of polymorphic automations and other convenience features outweigh the loss of this one enum.